src/java.base/macosx/native/libnio/fs/BsdNativeDispatcher.c

```diff
@@ -225,6 +227,22 @@ Java_sun_nio_fs_BsdNativeDispatcher_getmntonname0(JNIEnv *env, jclass this,
     return mntonname;
 }
 
+JNIEXPORT jint JNICALL
+Java_sun_nio_fs_BsdNativeDispatcher_clonefile0(JNIEnv* env, jclass this,
+    jlong srcAddress, jlong dstAddress, int flags)
```

Comment:
flags should be a "jint" (not int).

Reply:
Fixed in 2157a1480229fa697c824ae814f11a309627d768.